moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -108,21 +110,26 @@ void MotionPlanningFrame::pathConstraintsIndexChanged(int index)
 
 void MotionPlanningFrame::onClearOctomapClicked()
 {
-  std_srvs::Empty srv;
-  clear_octomap_service_client_.call(srv);
+  auto req = std::make_shared<std_srvs::srv::Empty::Request>();
+  auto result = clear_octomap_service_client_->async_send_request(req);
+
+  if (rclcpp::spin_until_future_complete(node_, result) != rclcpp::FutureReturnCode::SUCCESS)
```

Comment:
This will throw an exception since it's already added to an executor inside Rviz